pages/billing/troubleshooting/index.mdx

```diff
@@ -1,8 +1,59 @@
 ---
 meta:
-  title: &title Billing - Troubleshooting
-  description: &desc Billing Troubleshooting
+  title: Billing - Troubleshooting
+  description: Find troubleshooting resources for resolving issues with Scaleway Billing.
 content:
-  h1: *title
-  paragraph: *desc
----
\ No newline at end of file
+  paragraph: Find troubleshooting resources for resolving issues with Scaleway Billing.
+dates:
+  posted: 2023-10-05
+categories:
+  - billing
+---
+
+<ProductHeader
+  productName="Billing troubleshooting"
+  productLogo="accountBilling"
+  description="Discover how to solve the most common issues with Scaleway Billing"
+/>
+
+## Featured Pages
+
+<Grid>
+    <DefaultCard
+      title="Billing FAQ"
+      description="General info on Scaleway billing."
+      url="/faq/billing/"
+      label="See more"
+    />
+    <DefaultCard
+      title="How to add a payment method"
+      description="Step-by-step guide to adding a payment method to your Scaleway account."
```

Comment:
Same comment as on the other PR, I think most of this description will be cut off by the card, and there's no way to expand it